tests/test_checkout_caching.py

```diff
@@ -137,13 +137,13 @@ def test_google_cached_checkout_creates_multiregional_storage_type(self, mock_ch
     def test_google_uncached_checkout_creates_durable_storage_type(self, mock_check):
```

Comment:
nit: You could also update the test name, perhaps:
```
test_google_uncached_checkout_creates_uncached_storage_type
```

Reply:
fixed